rotkehlchen/chain/ethereum/decoding/decoder.py

```diff
@@ -363,7 +363,7 @@ def _maybe_decode_internal_transactions(
                 continue
 
             event_type, location_label, counterparty, verb = direction_result
-            preposition = 'to' if verb == 'Send' else 'from'
+            preposition = 'to' if event_type in {HistoryEventType.SPEND, HistoryEventType.TRANSFER} else 'from'  # noqa: E501
```

Comment:
Shouldn't `DEPOSIT` also be here? I believe it makes sense to create a constant with the types that are sending stuff out of the account and use it here and below and in the erc20/721 transfers

Reply:
Yeah, perhaps it makes sense, thanks